### DISCUSSION TITLE
Images in visual editor

### DISCUSSION THREAD

**Discussion by macjones321** [requarks/wiki#7076] [Help / Questions] [answered]
pretty new at this. I can't seem to get text to wrap around images in visual editor, or in the mardown editor. 

Was wondering if there is a way to do it in these editors instead of having to use the raw html editor.

Also are you limited to only right and left align? or is it somehow possible to move the image anywhere using the visual editor?

**Comment by NGPixel** [comment, marked as answer]
Not at the moment.
